Managers/FrameMan.cpp

```diff
@@ -510,7 +510,7 @@ namespace RTE {
 		}
 
 		// TODO: Remove this once GCC13 is released and switched to. std::format and std::chrono::time_zone are not part of latest libstdc++.
-#if _LINUX_OR_MACOSX_
+#if defined(__GNUC__) && __GNUC__ < 13
```

Comment:
is it possible to check using feature testing?

Reply:
no because gcc13 is missing 2 minor features from chrono that are required for the correct feature marker (really annoying)

Reply:
I'm not sure what you mean. Surely if they're missing the feature, it'll be marked as not using the feature?

Reply:
the `__cpp__lib__*` macros for feature testing aren't that granular and always require the _full_ set of library features on the set library date which gcc13 doesn't yet have (missing string stream parsers for calendar entries), and I don't know of another way to do feature tests

Reply:
I don't see what the issue with that is. That correctly means that if you feature test on this, it'll correctly say that gcc13 is missing these features, and it'll go down the alternate codepath. That's the desired behaviour, isn't it?

Reply:
yes, but all features that are used hear _are_ available on gcc13

Reply:
Oh I see what you mean. GCC13 doesn't have that, so it'd delay removal for another version. Hmm

Reply:
If GCC13 is already released, can we just upgrade to it and remove this mess here entirely?

Reply:
no because current ubuntu-lts (22.04) is stuck on gcc12, I'd like to always support the current ubuntu LTS since that's what most popular distros are based on. Next ubuntu LTS release is April 2024

Reply:
Sure, okay. All good then